exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SchemaUtils.kt

```diff
@@ -543,6 +543,7 @@ object SchemaUtils {
                 addMissingColumnsStatements(tables = tables, withLogs)
             }
             logTimeSpent("Executing alter table statements", withLogs) {
+                alterStatements.forEach { exposedLogger.info("Alter table statements to execute - $it") }
```

Comment:
This would print "Alter table statements to execute" next to each ALTER statement.

You could also call `addLogger(StdOutSqlLogger)` in your transaction before calling `SchemaUtils.createMissingTablesAndColumns` and that would log the ALTER statements (and all other SQL statements being executed). How about doing that instead of this?

Reply:
@joc-a 
Thank you for the review.
As you mentioned, developers need to be aware that they must include addLogger(StdOutSqlLogger) in the transaction block. Since the ALTER command modifies the table, how about logging based on the value of the withLogs flag?